examples/auto-scaling/rstudio-workbench/values.yaml

```diff
@@ -0,0 +1,173 @@
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/router.middlewares: "default-default-add-forwarded-headers@kubernetescrd"
+
+  hosts:
+    - host: hostname.mydomain.com
+      paths:
+        - /
+
+homeStorage:
+  create: true
+  storageClassName: efs-client
+
+service:
+  port: 80
+  annotations:
+    traefik.ingress.kubernetes.io/service.sticky.cookie: "true"
+    traefik.ingress.kubernetes.io/service.sticky.cookie.name: RSW-SESSION-COOKIE
+    traefik.ingress.kubernetes.io/service.sticky.cookie.secure: "true"
+    traefik.ingress.kubernetes.io/service.sticky.cookie.samesite: "none"
+    traefik.ingress.kubernetes.io/service.sticky.cookie.httponly: "true"
+
+image:
+  repository: "ghcr.io/rstudio/rstudio-workbench"
+  tag: "2021.09.0-351.pro6"
```

Comment:
let's remove this tag and repository (the whole `image` section) so this becomes more timeless 😄

Reply:
Its been nuked :)